src/Tests/StatsdUDPTests.cs

```diff
@@ -1,155 +1,155 @@
-using System;
-using System.Threading;
-using System.Collections.Generic;
-using NUnit.Framework;
-using StatsdClient;
-using Tests.Helpers;
+//using System;
+//using System.Threading;
+//using System.Collections.Generic;
+//using NUnit.Framework;
+//using StatsdClient;
+//using Tests.Helpers;
 
 
-namespace Tests
-{
-    // Most of StatsUDP is tested in StatsdUnitTests. This is mainly to test the splitting of oversized
-    // UDP packets
-    [TestFixture]
-    public class StatsUDPTests
-    {
-        private UdpListener udpListener;
-        private Thread listenThread;
-        private const int serverPort = 23483;
-        private const string serverName = "127.0.0.1";
-        private StatsdUDP udp;
-        private Statsd statsd;
-        private List<string> lastPulledMessages;
+//namespace Tests
+//{
+//    // Most of StatsUDP is tested in StatsdUnitTests. This is mainly to test the splitting of oversized
+//    // UDP packets
+//    [TestFixture]
+//    public class StatsUDPTests
```

Comment:
Good catch. I commented this out while making some changes and forgot to re-enable. Hold on this while I update tests and commit an update to the branch..